cpp/include/milvus-storage/packed/reader.h

```diff
@@ -70,6 +90,8 @@ class PackedRecordBatchReader : public arrow::RecordBatchReader {
   size_t memory_limit_;
   size_t buffer_available_;
   std::vector<std::unique_ptr<parquet::arrow::FileReader>> file_readers_;
+  std::vector<size_t> start_row_groups_;
```

Comment:
It would be much cleaner if these attributes go to some `FileRecordBatchReader`, because they make no sense in packed scenarios.